pytest-selenium/tests/test_toc.py

```diff
@@ -1,23 +1,62 @@
 from pages.content import Content
 from . import markers
 
-@markers.test_case('C250849')
+
+@markers.test_case("C250849")
 @markers.parametrize("book_slug,page_slug", [("college-physics", "preface")])
 @markers.nondestructive
 def test_toc_toggle_button_opens_and_closes(selenium, base_url, book_slug, page_slug):
+    """ Test that table of contents toggle button opens and closes the sidebar
+
+    The table of contents sidebar is open by default for desktop resolutions
+    and closed for mobile and tablet. We need to do different actions based
+    on the resolution. When the resolution is desktop we
+    click the toc button on the sidebar to close and the toc button on the
+    toolbar to open. It's the other way around for mobile and tablet
+    resolutions and the sidebar is closed by default.
+
+    """
     # GIVEN: The selenium driver, base_url, book_slug, and page_slug
 
     # WHEN: The book and page URL is loaded
-    #   AND: The table of contents toggle button is clicked
     content = Content(selenium, base_url, book_slug=book_slug, page_slug=page_slug).open()
 
-    toc = content.table_of_contents
-    content.click_table_of_contents_button()
+    toolbar = content.toolbar
+    sidebar = content.sidebar
+
+    # AND: Window width is 1024 or greater (Desktop)
+    if selenium.get_window_size()["width"] > 1024:
```

Comment:
magic numbers should be defined somewhere, maybe a top level region with a `isDesktop()` or `isMobile()`?